code/modules/surgery/organs/lavaland.dm

```diff
@@ -0,0 +1,367 @@
+/obj/item/organ/vocal_cords/colossus
+	name = "divine vocal cords"
+	icon_state = "voice_of_god"
+	zone = "mouth"
+	slot = "vocal_cords"
+	actions_types = list(/datum/action/item_action/organ_action/colossus)
+	var/next_command = 0
+	var/cooldown_stun = 900
+	var/cooldown_damage = 600
+	var/cooldown_meme = 300
+	var/cooldown_none = 150
+	var/base_multiplier = 1
+	spans = list("colossus","yell")
+
+/datum/action/item_action/organ_action/colossus
+	name = "Voice of God"
+	var/obj/item/organ/vocal_cords/colossus/cords = null
+
+/datum/action/item_action/organ_action/colossus/New()
+	..()
+	cords = target
+
+/datum/action/item_action/organ_action/colossus/IsAvailable()
+	if(world.time < cords.next_command)
+		return FALSE
+	if(!owner)
+		return FALSE
+	if(!owner.can_speak())
+		return FALSE
+	if(check_flags & AB_CHECK_CONSCIOUS)
+		if(owner.stat)
+			return FALSE
+	return TRUE
+
+/datum/action/item_action/organ_action/colossus/Trigger()
+	. = ..()
+	if(!IsAvailable())
+		if(world.time < cords.next_command)
+			owner << "<span class='notice'>You must wait [(cords.next_command - world.time)/10] seconds before Speaking again.</span>"
+		return
+	var/command = stripped_input(owner, "Speak with the Voice of God", "Command", max_length = 140)
+	if(!command)
+		return
+	owner.say(".x[command]")
+
+/obj/item/organ/vocal_cords/colossus/can_speak_with()
+	if(world.time < next_command)
+		owner << "<span class='notice'>You must wait [(next_command - world.time)/10] seconds before Speaking again.</span>"
+		return FALSE
+	if(!owner)
+		return FALSE
+	if(!owner.can_speak())
+		owner << "<span class='warning'>You are unable to speak!</span>"
+		return FALSE
+	if(owner.stat)
+		return FALSE
+	return TRUE
+
+/obj/item/organ/vocal_cords/colossus/speak_with(message)
```

Comment:
According to your use of `speak_with`, shouldn't the message you speak be returned?